cluster_config/configurations.yaml.tpl

```diff
@@ -37,10 +37,10 @@ Configurations:
     "hive.compactor.initiator.on": "true"
     "hive.compactor.worker.threads": "1"
     "hive.support.concurrency": "true"
-    "javax.jdo.option.ConnectionURL": "jdbc:mysql://${hive_metastore_fqdn}:3306/hive?createDatabaseIfNotExist=true"
+    "javax.jdo.option.ConnectionURL": "jdbc:mysql://${hive_metastore_endpoint}:3306/${hive_metastore_database_name}?createDatabaseIfNotExist=true"
```

Comment:
you have to pass the value for this to this file otherwise it wont resolve correctly.